<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MdllSetup</class>
 <widget class="QDialog" name="MdllSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mdll setup</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="textLabel1_4_2">
       <property name="text">
        <string>MCPD ID</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="MCPDSpinBox" name="devid">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximum">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="dataTab">
      <attribute name="title">
       <string>Data</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="toolTip">
          <string>select transmitted data</string>
         </property>
         <property name="title">
          <string>Dataset</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QRadioButton" name="posButton">
            <property name="toolTip">
             <string>Energy, X and Y coordinate</string>
            </property>
            <property name="text">
             <string>E, X, Y (default)</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="timingButton">
            <property name="toolTip">
             <string>E, and timing sums for X and Y</string>
            </property>
            <property name="text">
             <string>E, tX, tY</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>87</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="setDataset">
            <property name="text">
             <string>Set Dataset</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="toolTip">
          <string>software &quot;window discriminator&quot; for timing sums</string>
         </property>
         <property name="title">
          <string>Timing Window</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>tsum X</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QSpinBox" name="txlo">
            <property name="toolTip">
             <string>min 100, max 1000</string>
            </property>
            <property name="minimum">
             <number>100</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="txhi">
            <property name="toolTip">
             <string>min 100, max 1000</string>
            </property>
            <property name="minimum">
             <number>100</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>tsum Y</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QSpinBox" name="tylo">
            <property name="toolTip">
             <string>min 100, max 1000</string>
            </property>
            <property name="minimum">
             <number>100</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="tyhi">
            <property name="toolTip">
             <string>min 100, max 1000</string>
            </property>
            <property name="minimum">
             <number>100</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QPushButton" name="setTimingWindow">
            <property name="text">
             <string>Set timing</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="toolTip">
          <string>software &quot;window discriminator&quot; for energy</string>
         </property>
         <property name="title">
          <string>Energy Window</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Lower Threshold</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QSpinBox" name="elo">
            <property name="toolTip">
             <string>min 0, max 255</string>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="value">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Upper Threshold</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QSpinBox" name="ehi">
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="value">
             <number>240</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="setEnergyWindow">
            <property name="text">
             <string>Set  E Window</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="hardwareTab">
      <attribute name="title">
       <string>Hardware</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="toolTip">
          <string>Offset on timing delay, shifts spectrum
Range of timing delay, scales spectrum</string>
         </property>
         <property name="title">
          <string>Delay</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Offset</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="shiftX">
            <property name="toolTip">
             <string>min 0, max 255, arb. units</string>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label9">
            <property name="text">
             <string>Y</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="shiftY">
            <property name="toolTip">
             <string>min 0, max 255, arb. units</string>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Range</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="scaleX">
            <property name="toolTip">
             <string>min 0, max 255, arb. units</string>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="value">
             <number>48</number>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label8">
            <property name="text">
             <string>Y</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QSpinBox" name="scaleY">
            <property name="toolTip">
             <string>min 0, max 255, arb. units</string>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="value">
             <number>48</number>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="2">
           <widget class="QPushButton" name="setDelay">
            <property name="text">
             <string>Set Delay</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="toolTip">
          <string>set Thresholds for X, Y and Anode CFDs</string>
         </property>
         <property name="title">
          <string>CFD Thresholds</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QSpinBox" name="threshX">
            <property name="toolTip">
             <string>min 0, max 255, arb. units</string>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="value">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="shiftY_3">
            <property name="text">
             <string>Y</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QSpinBox" name="threshY">
            <property name="toolTip">
             <string>min 0, max 255, arb. units</string>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="value">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Anode</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QSpinBox" name="threshA">
            <property name="toolTip">
             <string>min 0, max 255, arb. units</string>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="value">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="2">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>54</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="0" colspan="3">
           <widget class="QPushButton" name="setCfd">
            <property name="text">
             <string>Set CFDs</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>163</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MCPDSpinBox</class>
   <extends>QSpinBox</extends>
   <header>MCPDSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>MdllSetup</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>setTimingWindow</sender>
   <signal>clicked()</signal>
   <receiver>MdllSetup</receiver>
   <slot>setTimingSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>setEnergyWindow</sender>
   <signal>clicked()</signal>
   <receiver>MdllSetup</receiver>
   <slot>setEnergySlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>417</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>setCfd</sender>
   <signal>clicked()</signal>
   <receiver>MdllSetup</receiver>
   <slot>setThresholdsSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>419</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>setDelay</sender>
   <signal>clicked()</signal>
   <receiver>MdllSetup</receiver>
   <slot>setSpectrumSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>setDataset</sender>
   <signal>clicked()</signal>
   <receiver>MdllSetup</receiver>
   <slot>setDatasetSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>90</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>devid</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MdllSetup</receiver>
   <slot>displaySlot(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>1</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>displaySlot(int)</slot>
  <slot>setMCPD(int)</slot>
  <slot>setThresholdsSlot()</slot>
  <slot>setTimingSlot()</slot>
  <slot>setEnergySlot()</slot>
  <slot>setSpectrumSlot()</slot>
  <slot>setDatasetSlot()</slot>
  <slot>setPulserSlot()</slot>
 </slots>
</ui>
